test/apiv2.test.js

```diff
@@ -25,6 +26,23 @@ describeOrSkip("API V2", function () {
     await migrateDatabase("max", pool);
   });
 
+  it("serves the version path", async function () {
+    process.env.RUN_ID = "buildinfo";
+    process.env.GIT_SHA = "commitinfo";
+    const app = await createServer();
+    const response = await app.inject({
+      method: "GET",
+      url: "/api/v2/version",
+    });
+    assert.equal(response.statusCode, 200);
+    const j = response.json();
+    const p = JSON.parse(fs.readFileSync("package.json", "utf8"));
+    assert.equal(j.version, p.version);
+    assert.equal(j.commit, "commitinfo");
+    assert.equal(j.build, "buildinfo");
+    assert.equal(j.source, "https://github.com/mdn/mdn-http-observatory");
```

Comment:
Note that you should also be able to check it like this:

```suggestion
    assert.deepEqual(j, {
        version: p.version,
        commit: "commitinfo",
        build: "buildinfo",
        source: "https://github.com/mdn/mdn-http-observatory"
    };
```

The advantage is that in case of failure, the error message should be more descriptive, rather than just saying that the assertion failed in line x.